infra/base-images/base-runner/coverage

```diff
@@ -31,11 +31,12 @@ COVERAGE_OUTPUT_DIR=${COVERAGE_OUTPUT_DIR:-$OUT}
 
 DUMPS_DIR="$COVERAGE_OUTPUT_DIR/dumps"
 FUZZER_STATS_DIR="$COVERAGE_OUTPUT_DIR/fuzzer_stats"
+COVERAGE_REPORT_DIR="$COVERAGE_OUTPUT_DIR/covreports"
```

Comment:
I have an annoying idea. Maybe we should call these reports textcov to distinguish them from the HTML reports.
I think that's what they are right?
If you don't want to change the .covreport suffix, maybe just change the directory name (and variable name).
WDYT?

Reply:
well, yes they are the output of llvm-cov show. Made changes.